README.md

```diff
@@ -29,6 +29,20 @@ The steps for contributing changes are the following,
 
 Please make sure your code is well organized and commented before opening the pull request.
 
+## PR labeling strategy
+
+On a daily basis, all PRs should be labeled with one of the review process tags:
+- `PR: Draft`
+- `PR: Needs Review`
+- `PR: Work Needed`
+- `PR: Mergeable`
+
+The person opening the PR should apply `PR: Draft` until the work is finished, then the person can apply the `PR: Needs Review` label as soon the PR is ready to be reviewed.
+The reviewer can change it to `PR: Mergeable` or `PR: Work Needed` if the PR needs more work and assign it back to the corresponding developer.
+
+Please respond to review requests in a timely manner, or indicate another reviewer would be better. Code review is everyone’s responsibility.
```

Comment:
Please respond to review requests in a timely manner, or indicate if another reviewer would be better. Code review is everyone’s responsibility.